validation/data-availability-type_test.go

```diff
@@ -33,22 +34,25 @@ func testDataAvailabilityType(t *testing.T, chain *ChainConfig) {
 	batchInboxAddress := chain.BatchInboxAddr
 	require.NotZero(t, batchInboxAddress)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	defer cancel()
-
 	depth := chain.SequencerWindowSize
 	require.NotZero(t, depth)
 
-	// First attempt without needing an archive node
-	//  - full nodes keep the last 128 blocks but we use 120 to give a buffer
-	nonArchivalDepth := 120
-	blockNum, found, err := eth.CheckRecentTxs(ctx, client, nonArchivalDepth, common.Address(batchSubmitterAddress))
-	if !found {
-		if err != nil {
-			t.Log("failed to check recent batcher txs, will attempt retry")
+	blockNum, found, err := retry.Do2(context.Background(), 3, retry.Exponential(), func() (uint64, bool, error) {
```

Comment:
Should we extract the number of retries into a constant, maybe even shared across the many tests which hit the same endpoint?

Reply:
Done in https://github.com/ethereum-optimism/superchain-registry/pull/682/commits/aca3df43cb47b1bb7e7e28a512de69b075a800cd